app/dashboard/gallery/orders/page.tsx

```diff
@@ -1,6 +1,18 @@
+"use client";
+import { useState } from "react";
 import NoOrder from "./components/NoOrder";
-export default function page() {
+import OrderTab from "./components/OrderTab";
+import Pending from "./components/PendingPage";
+import HistoryPage from "./components/HistoryPage";
+
+
+export default function Orders() {
+  const[tab, setTab] = useState("pending");
   return (
-    <NoOrder />
+    <div>
+     <OrderTab tab={tab} setTab={setTab} />
+     <div>{tab === "pending" ? <Pending /> : <HistoryPage />}</div>
```

Comment:
This wrapper div serves no purpose, so take it out and just conditionally render the Pending and history pages. Don't forget, these pages are div wrappers themselves. So you'd just be having unused div element in the DOM.

Reply:
I used React Fragments to wrap instead. Done